Bottom: 72cdc2af69ebdaf1abaf38e6ea20f9062b8e26fc
Top:    3d1870b85b2e6ce2b93cc79e722cb7045b891243
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 00:59:09 +0100

Refresh of action-c-remove-commented-out-

---

diff --git a/src/action.c b/src/action.c
index b83325b..59e8139 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2748,7 +2748,6 @@ ActionDisplay (int argc, char **argv, int childX, int childY)
 	  break;
 
 	case F_CycleCrosshair:
-	  //Crosshair.shape++;
 	  Crosshair.shape = CrosshairShapeIncrement(Crosshair.shape);
 	  if (Crosshair_Shapes_Number == Crosshair.shape)
 	    Crosshair.shape = Basic_Crosshair_Shape;
